### PR TITLE
KFSTP-1870 Move parms to AR from CG

### PR DESCRIPTION
**Pull request by kkronenb**

### CODE REVIEW

File: work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java [hunk 1]
```diff
@@ -160,6 +164,30 @@ public boolean isContractsGrantsBillingEnhancementActive() {
         return getConfigurationService().getPropertyValueAsBoolean(KFSConstants.CONTRACTS_GRANTS_BILLING_ENABLED);
     }
 
+    public String getDefaultDunningCampaignCode() {
+        String parameterDunningCampaignCode = "";
+        if (getParameterService().parameterExists(DunningCampaign.class, ArConstants.DEFAULT_DUNNING_CAMPAIGN_PARAMETER)) {
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java]: Do we need this anymore?  Can't we just send back the empty string as a default?

File: work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java [hunk 2]
```diff
@@ -160,6 +164,30 @@ public boolean isContractsGrantsBillingEnhancementActive() {
...
+            parameterDunningCampaignCode = getParameterService().getParameterValueAsString(DunningCampaign.class, ArConstants.DEFAULT_DUNNING_CAMPAIGN_PARAMETER);
+        }
+        return parameterDunningCampaignCode;
+    }
+
+    public String getDefaultBillingFrequency() {
+        String parameterDefaultBilling = "";
+        if (getParameterService().parameterExists(ContractsGrantsInvoiceDocument.class, ArConstants.DEFAULT_BILLING_FREQUENCY_PARAMETER)) {
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java]: see above

File: work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java [hunk 3]
```diff
@@ -160,6 +164,30 @@ public boolean isContractsGrantsBillingEnhancementActive() {
...
+            parameterDefaultBilling = getParameterService().getParameterValueAsString(ContractsGrantsInvoiceDocument.class, ArConstants.DEFAULT_BILLING_FREQUENCY_PARAMETER);
+        }
+        return parameterDefaultBilling;
+    }
+
+    public String getDefaultInvoicingOption() {
+        String parameterDefaultInvoicingOption = "";
+        if (getParameterService().parameterExists(ContractsGrantsInvoiceDocument.class, ArConstants.DEFAULT_INVOICING_OPTION_PARAMETER)) {
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java]: see above

File: work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java (hunk 1 above)
Comment (kkronenb) [inline work/src/org/kuali/kfs/module/ar/service/impl/AccountsReceivableModuleBillingServiceImpl.java]: probably not, I'll clean it up

**Pull request merged**